### DISCUSSION TITLE
PR and issues templates

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#65] [General]
We could still add PR and issues templates for the release if we want. 
I had made an attempt in Foodme, but there are other great templates out there.

What do you think?

**Comment by marchoeppner** [comment]
In principle yes; I wouldn't see it as an absolute "must" for 1.0 - but if you have something that is more or less "ready-to-use", by all means,

**Comment by gregdenay** [comment]
I added some templates in [.github](https://github.com/bio-raum/FooDMe2/tree/dev/.github), these will only take effect after we merged `dev` in `main`.
It's a best try, we can still modify them with a quick commit on the main branch later. No need for a release for this kind of things in my opinion.
